spec/lib/tasks/delete_activity_spec.rb

```diff
@@ -145,5 +145,13 @@
 
       expect { delete_activity(activity_id: activity.id) }.to change { Activity.count }.by(-2)
     end
+
+    it "deletes the descendant acitivities" do
```

Comment:
`<nit-pick alert>` typo in "acitivities"

Reply:
Not a nit pick! I need the spelling help! 👍